e2e_samples/parking_sensors_synapse/.devcontainer/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
```

Comment:
This seems to have doubled up the .devcontainer. That is there is a .devcontainer folder inside the .devcontainer. Probably delete this one?